### PR TITLE
Add Different Reusable Components (excacomp)

### PR DESCRIPTION
**Pull request by Maddoxx88**

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -142,5 +142,32 @@
...
+    "preview": "excacomp/mobile-kit.png",
+    "date": "2021-01-17"
+  },
+  {
+    "name": "Pricing Component",
+    "description": "A modern pricing component commonly used in websites.",
+    "author": "Sunit Shirke",
+    "url": "https://github.com/Maddoxx88",
+    "source": "excacomp/pricing-component.excalidrawlib",
+    "preview": "excacomp/pricing-component.png",
```
Comment (lipis) [inline libraries.json:161]:
```suggestion
    "preview": "excacomp/pricing-component.jpg",
```

**Comment by lipis** [comment]
Could you change all the previous to use PNG?

**Comment by lipis** [comment]
Also could we possible put them all in one.. they are all web components

**Comment by Maddoxx88** [comment]
Merged all the web components into one library and all previews are in PNG @lipis

**Comment by lipis** [comment]
Can you also update the entries in the `libraries.json`? There should be one lib and one png file

**Comment by lipis** [comment]
You can always check the latest preview here: https://excalidraw-libraries-git-fork-maddoxx88-main.excalidraw.vercel.app

**Comment by Maddoxx88** [comment]
Done @lipis

File: libraries.json [hunk 2]
```diff
@@ -142,5 +142,23 @@
...
-  }
+  },
+  {
+    "name": "Mobile Kit",
+    "description": "A collection of commonly used mobile screens which you can drag and drop into Excalidraw.",
+    "author": "Sunit Shirke",
+    "url": "https://github.com/Maddoxx88",
+    "source": "excacomp/mobile-kit.excalidrawlib",
+    "preview": "excacomp/mobile-kit.png",
+    "date": "2021-01-17"
```
Comment (lipis) [inline libraries.json:153]:
```suggestion
    "date": "2021-01-20"
```

File: libraries.json [hunk 3]
```diff
@@ -142,5 +142,23 @@
...
+  },
+  {
+    "name": "Web Kit",
+    "description": "A collection of commonly used web components which you can drag and drop into Excalidraw.",
+    "author": "Sunit Shirke",
+    "url": "https://github.com/Maddoxx88",
+    "source": "excacomp/web-kit.excalidrawlib",
+    "preview": "excacomp/web-kit.png",
+    "date": "2021-01-17"
```
Comment (lipis) [inline libraries.json:162]:
```suggestion
    "date": "2021-01-20"
```

File: libraries.json [hunk 4]
```diff
@@ -142,5 +142,23 @@
     "source": "pclainchard/it-logos.excalidrawlib",
     "preview": "pclainchard/it-logos.png",
     "date": "2021-01-12"
-  }
+  },
+  {
+    "name": "Mobile Kit",
+    "description": "A collection of commonly used mobile screens which you can drag and drop into Excalidraw.",
```
Comment (lipis) [inline libraries.json:148]:
```suggestion
    "description": "A collection of commonly used mobile screens.",
```

File: libraries.json [hunk 5]
```diff
@@ -142,5 +142,23 @@
...
+    "author": "Sunit Shirke",
+    "url": "https://github.com/Maddoxx88",
+    "source": "excacomp/mobile-kit.excalidrawlib",
+    "preview": "excacomp/mobile-kit.png",
+    "date": "2021-01-20"
+  },
+  {
+    "name": "Web Kit",
+    "description": "A collection of commonly used web components which you can drag and drop into Excalidraw.",
```
Comment (lipis) [inline libraries.json:157]:
```suggestion
    "description": "A collection of commonly used web components.",
```

**Pull request merged**